test/lint/codespell.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh -eu
-codespell -S 'internal/server/apparmor/*.profile.go' -S 'doc/html/*' -I .codespell-ignore client cmd doc grafana internal scripts shared test
+codespell -S 'internal/server/apparmor/*.profile.go' -S 'doc/html/*' -I .codespell-ignore client cmd doc grafana internal scripts shared test --ignore-regex '"nd"'
```

Comment:
Instead we should ignore all of `doc/reference/manpages/*` as that's also generated content.

Reply:
I've made the change